composables/useTimePopup.ts

```diff
@@ -0,0 +1,25 @@
+export function useOneTimePopup (key: string) {
+  const show = ref(false)
+
+  onMounted(() => {
+    try {
+      if (!localStorage.getItem(key)) {
```

Comment:
consider using `https://vueuse.org/core/useStorage/`

Reply:
[♻️ Use useStorage for state management](https://github.com/likecoin/nft-book-press/pull/367/commits/6706dcbaab20edb0e8fd5791980dd6acaa00780e)